### DISCUSSION TITLE
Searching while not signed in gives no useful error message, it just says it can't find the string

### DISCUSSION THREAD

**Discussion by james-martens** [community/community#69226] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I tried to search a repo for a string while not signed in. Instead of telling me that search doesn't work when not signed in it just said that there were no results. Eventually I tried to search for a specific symbol by clicking a function name in one of the files, and only then was I told that I need to sign in for searching to work. And I'm not sure if this is relevant, but once signed in I was told that it needed to index the repo first before searching would work (which I wasn't told when not signed in). Needless to say I think this is very confusing interface design, and in my case it caused a lot of frustration.

**Comment by shourovr82** [comment]
you can use web editor by clicking dot (  .  ) on your keyboard , while viewing the repository. then you can find your string or something by clicking ctrl+shift+f    

if not work let me know.
